addons/website_sale/data/mail_template_data.xml

```diff
@@ -31,7 +31,7 @@
                                         <strong t-out="line.product_id.display_name or ''">[FURN_7800] Desk Combination</strong><br/><t t-out="line.name or ''">[FURN_7800] Desk Combination Desk combination, black-brown: chair + desk + drawer.</t>
                                     </td>
                                     <td width="100px" align="right">
-                                        <t t-out="int(line.product_uom_qty) or ''">10000</t> <t t-out="line.product_uom.name or ''">Units</t>
+                                        <t t-out="int(line.product_uom_qty) or ''">10000</t> <t t-out="line.product_uom_id.name or ''">Units</t>
```

Comment:
Not fully sure this is handled during the upgrade script, since the mail template is noupdate, maybe you have to call a specific upgrade util to handle this change in the existing databases (unless it's already handled by `rename_field`).

Reply:
@aj-fuentes Do this will handle via `rename_field` ? or we have to call `update_record_from_xml`  in script?

Reply:
No this is NOT handled, you need to force the update for any template that is `noupdate=True`. Maybe edit all mail templates (in case some were duplicated or manually done).